CHANGELOG

```diff
@@ -1,3 +1,39 @@
+Release 0.40.0
+
+- Multi-target support
+- Update license to Apache 2.0
+- Add option --collector.dbstatsfreestorage to receive freeStorage stats from dbstats
+- New Current Op Metrics collector
+- New collector to collect data from the system profile
+- Add support for timestamp metrics with seconds precision
+- Update Go version
+- Update dependencies
+
+Release 0.39.0
+
+- Fixed issue with connection to authenticated MongoDB Arbiter
+- Update dependencies
+
+Release 0.38.0
+
+- Fixed setting auth options for connection to MongoDB
+
+Release 0.37.0
+
+- Disable creating metric mongodb_mongod_storage_engine when it's not possible to retrieve (for example for mongos type of instance)
+- Update dependencies
+
+Release 0.36.0
+
+- Added security metrics with info about encryption
+- Fixed fetching build info for mongos instance
+- Fix metric mongodb_mongos_sharding_balancer_enabled
+
+Release 0.35.0
+
+- Added to metrics info about running MongoDB version and Mongo vendor
+- Removed unused collector
+
 Release 0.34.0-bugsnag1
```

Comment:
We probably should add a `Release 0.40.0-bugsnag1` as well

Reply:
Added.